lib/index.js

```diff
@@ -20,7 +18,7 @@ function jsonClient(baseUrl, options) {
 
 	return function jsonClientRequest(method, path, params, body, options) {
 		const query = params ? '?' + qs.stringify(params) : '';
-		const resolved = new url(path + query, resolvedBaseUrl);
+		const resolved = new URL(path + query, resolvedBaseUrl);
```

Comment:
How sure are we on the compatibility of `url` or `URL`?

Reply:
that signature is for the node package and was only working  because of a conflict, the `url` npm package is not compatible with this

Reply:
old package: https://www.npmjs.com/package/url
node package: https://nodejs.org/api/url.html#url_new_url_input_base